upload_to_s3.py

```diff
@@ -29,11 +29,104 @@
-    "units": "select DEPT_UNIT_ID, DEPT_ID, DEPT, UNIT, UNIT_LONG_NAME, UNIT_SHORT_NAME, DEPT_UNIT_STATUS from lu_dept_units WHERE DEPT in(2400,6207,2507)",
-    "objects": "select OBJ_ID, OBJ_CLASS_ID, OBJ_CATEGORY_ID, OBJ_TYPE_ID, OBJ_GROUP_ID, OBJ_CODE, OBJ_LONG_NAME, OBJ_SHORT_NAME, OBJ_DESC, OBJ_REIMB_ELIG_STATUS, OBJ_STATUS, ACT_FL from lu_obj_cd",
-    "master_agreements": "select DOC_CD, DOC_DEPT_CD, DOC_ID, DOC_DSCR, DOC_PHASE_CD, VEND_CUST_CD, LGL_NM from DEPT_2400_MA_VW",
-    "fdus": "select * from ATD_SUBPROJECT_FDU_VW",
+    "task_orders": """
+    SELECT
+        atd_tk.TASK_ORDER_DEPT,
+        atd_tk.TASK_ORDER_ID,
+        atd_tk.TASK_ORDER_DESC,
+        atd_tk.TASK_ORDER_STATUS,
+        atd_tk.TASK_ORDER_TYPE,
+        atd_tk.TK_CURR_AMOUNT,
+        atd_tk.CHARGED_AMOUNT,
+        atd_tk.TASK_ORDER_BAL,
+        atd_tk.TASK_ORDER_ESTIMATOR,
+        buyer_tk.BYR_FDU
+    FROM
+        DEPT_2400_TK_VW atd_tk
+        LEFT JOIN ( SELECT DISTINCT
+                TASK_ORD_CD,
+                BYR_FDU
+            FROM
+                REL_BUYER_SELLER_FDU_TK) buyer_tk ON atd_tk.TASK_ORDER_ID = buyer_tk.TASK_ORD_CD
+    WHERE
+        TASK_ORDER_STATUS IS NOT NULL
+    """,
+    "units": """
+    SELECT
+        DEPT_UNIT_ID,
+        DEPT_ID,
+        DEPT,
+        UNIT,
+        UNIT_LONG_NAME,
+        UNIT_SHORT_NAME,
+        DEPT_UNIT_STATUS
+    FROM
+        lu_dept_units
+    WHERE
+        DEPT in(2400, 6207, 2507)
+	""",
+    "objects": """
+    SELECT
+        OBJ_ID,
+        OBJ_CLASS_ID,
+        OBJ_CATEGORY_ID,
+        OBJ_TYPE_ID,
+        OBJ_GROUP_ID,
+        OBJ_CODE,
+        OBJ_LONG_NAME,
+        OBJ_SHORT_NAME,
+        OBJ_DESC,
+        OBJ_REIMB_ELIG_STATUS,
+        OBJ_STATUS,
+        ACT_FL
+    FROM
+        lu_obj_cd
+	""",
+    "master_agreements": """
+    SELECT
+        DOC_CD,
+        DOC_DEPT_CD,
+        DOC_ID,
+        DOC_DSCR,
+        DOC_PHASE_CD,
+        VEND_CUST_CD,
+        LGL_NM
+    FROM
+        DEPT_2400_MA_VW
+    """,
+    "fdus": """
+    SELECT
+        DEPT_CODE_NAME,
+        SUB_PROJECT_ID,
+        SUBPROJECT_ID_UK,
+        SP_NUMBER_TXT,
+        FDU_ID,
+        FDU,
+        FUND,
+        FUNDNAME,
+        DEPT,
+        DEPT_ID,
+        DEPT_UNIT_ID,
+        DEPT_UNIT_STATUS,
+        UNIT,
+        UNIT_LONG_NAME,
+        UNIT_SHORT_NAME
+    FROM
+        ATD_SUBPROJECT_FDU_VW
+	""",
+    "subprojects": """
+        SELECT
+        PROJECT_NUMBER,
+        SP_NUMBER_TXT,
+        SP_NAME,
+        SP_DESCRIPTION,
+        SP_DETAILED_SCOPE,
+        SUB_PROJECT_MANAGER,
+        SUB_PROJECT_MANAGING_DEPT,
+        SP_STATUS
+    FROM
+        MSTR_IA_DEV.DEPT_2400_SUBPRJ_VW
+	""",
```

Comment:
new query

Reply:
thanks for linting these queries: it might be time to move them to a separate file?